product_docs/docs/pgd/5/reference/nodes-management-interfaces.mdx

```diff
@@ -285,15 +285,15 @@ bdr.create_node_group(node_group_name text,
 -   `parent_group_name` &mdash; If a node subgroup is being created, this must be the
       name of the parent group. Provide `NULL` (the default) when creating the main
       node group for the cluster.
--   `join_node_group` &mdash; This parameter determines whether the node should
-      join the group being created. The default value is `true`. `false` may
-      be provided when creating a subgroup which the local node will not join.
-      In this case, `parent_group_name` must be specified.
+-   `join_node_group` &mdash; This parameter determines whether the node
+      joins the group being created. The default value is `true`. You can provide `false`
```

Comment:
Providing `false` when creating a subgroup means the local node won't join the new group, for example, when creating a independent remote group.

Reply:
Used your text here.